distributed/scheduler.py

```diff
@@ -7471,7 +7472,11 @@ def _reevaluate_occupancy_worker(state: SchedulerState, ws: WorkerState):
 @cfunc
 @exceptval(check=False)
 def decide_worker(
-    ts: TaskState, all_workers, valid_workers: set, objective
+    ts: TaskState,
+    all_workers: set,
+    valid_workers: Optional[set],
```

Comment:
Cython doesn't handling `typing` objects like `Optional` yet. Though it already happily accepts `None` and checks for that

```suggestion
    valid_workers: set,
```

Reply:
I added optional mostly for type checking, e.g. in IDEs. is cython having trouble with this or does it simply ignore it?